glossary.yml

```
@@ -5892,7 +5892,15 @@
 
 
 - slug: test_data
-
+  ref:
+    - machine_learning
+  en:
+    term: "test data"
+    def: >
+      Test data is a portion of a dataset used to evaluate the correctness of a 
+      [machine learning](#machine_learning) algorithm after it has been trained. It should always
+      be separated from the [training data](#training_data) to ensure that the model is 
+      properly tested with unseen data.
```

```suggestion
      [machine learning](#machine_learning) algorithm after it has been trained.
      It should always be separated from the [training data](#training_data) to
      ensure that the model is properly tested with unseen data.
```

Changing line width.